### PR TITLE
Update PRR description based on community feedback

### PR DESCRIPTION
**Pull request by johnbelamaric**
This updates the production readiness process based upon the discussion in the latest leads meeting.

/assign @wojtek-t @deads2k @mrbobbytables 
/cc @ehashman @kubernetes/enhancements 

/hold for lazy consensus until 4pm Pacific time, January 20th.

### CODE REVIEW

**Comment by justaugustus** [comment]
Explicit hold for review by @kubernetes/enhancements-maintainers.
/hold

File: sig-architecture/production-readiness.md
Comment (wojtek-t) [inline sig-architecture/production-readiness.md:12]: nit: we were doing dry-runs also in 1.18, and I think the very first pilot phase was even from 1.17?

File: sig-architecture/production-readiness.md
Comment (wojtek-t) [inline sig-architecture/production-readiness.md:21]: That is potentially something we want to discuss. While targeting release should definitely imply PRR approval, I'm not convinced about moving to implementable.
I've seen multiple cases where KEPs moved to implementable was staying there for quite some time with not much traction so I'm not sure we want to necessary (a) all load to PRR team, (b) block authors from having that merged...

File: sig-architecture/production-readiness.md
Comment (wojtek-t) [inline sig-architecture/production-readiness.md:38]: Do we want to mention that we may want to do some internal shuffling within the PRR team if someone will get overloaded by this?

File: sig-architecture/production-readiness.md
Comment (wojtek-t) [inline sig-architecture/production-readiness.md:92]: This may be a better example: https://github.com/kubernetes/enhancements/pull/2179/files

File: sig-architecture/production-readiness.md
Comment (wojtek-t) [inline sig-architecture/production-readiness.md:47]: @deads2k @johnbelamaric - comment to ourselves
IIRC kepctl tool is based on the "prr-approver" field in the kep.yaml.
With our move towards "keps/prod-readiness/<kep-number>.yaml", I would like to remove prr-approver field (and it's also not reflecting the state-of-the-world, so we should probably update kepctl.

File: sig-architecture/production-readiness.md
Comment (johnbelamaric) [reply sig-architecture/production-readiness.md:21]: it's "target release && implementable", rather than just "target release". It's actually mandatory to move to "implementable" to be in the release, per the Enhancements team (see the KEP template).

Without checking for "implementable", KEPs that are still in "provisional" status can sometimes run into the CI validation.

File: sig-architecture/production-readiness.md
Comment (johnbelamaric) [reply sig-architecture/production-readiness.md:47]: so the assignment isn't made until the "keps/prod-readiness/.yaml" file is created? That's possible OK...but how do we do early reviews?

File: sig-architecture/production-readiness.md
Comment (wojtek-t) [reply sig-architecture/production-readiness.md:21]: So given the fact that you need to be "implementable" to target a release - do we need to mention this explicitly?

File: sig-architecture/production-readiness.md
Comment (wojtek-t) [reply sig-architecture/production-readiness.md:47]: I think that "prr-approver" in kep.yaml is now a bit redundant at this point.

I think for early reviews - we do them the same as we do:
- people who know who to assign - they will also have a PR with keps/prod-readiness/.yaml open
- people who don't know - we will be doing that based on query "no PR opened that adds that file"?

File: sig-architecture/production-readiness.md
Comment (johnbelamaric) [reply sig-architecture/production-readiness.md:21]: No, good point.

File: sig-architecture/production-readiness.md
Comment (johnbelamaric) [reply sig-architecture/production-readiness.md:12]: yup, you are right 1.17

File: sig-architecture/production-readiness.md
Comment (johnbelamaric) [reply sig-architecture/production-readiness.md:38]: done

File: sig-architecture/production-readiness.md
Comment (johnbelamaric) [reply sig-architecture/production-readiness.md:47]: ok, this is not something relevant for this PR, we will fix it when we change that

File: sig-architecture/production-readiness.md
Comment (johnbelamaric) [reply sig-architecture/production-readiness.md:92]: done

File: sig-architecture/production-readiness.md
Comment (ehashman) [inline sig-architecture/production-readiness.md:77]: Should this be "at least one release"? If we move to 3 releases/yr it will take 8mo of shadowing to make someone an approver, which seems unusually high for k8s, especially on a brand new team

**Comment by johnbelamaric** [comment]
I think that's fine. It will still be up to the existing approvers whether
they think someone is ready.

**Review by ehashman — APPROVED** [review]
Generally this LGTM, I think we've passed the lazy consensus deadline.

/lgtm

**Comment by coderanger** [comment]
/lgtm

Would you be interested in working with contributor-comms to put out a call for more members?

**Comment by johnbelamaric** [comment]
sure, what needs to be done?

**Comment by mrbobbytables** [comment]
/lgtm
The additional details do a good job of enumerating and explaining the concerns that were brought up in the chair/tl meeting.

**Comment by coderanger** [comment]
Just stop by #sig-contribex in Slack some time and we can work out the best option. Probably just a tweet? But we have a lot options now so depends on what audience you want to reach :)

File: sig-architecture/production-readiness.md
Comment (wojtek-t) [reply sig-architecture/production-readiness.md:47]: Sure - that was more to us.

**Review by wojtek-t — COMMENTED** [review]
/lgtm

Thanks!

**Review by justaugustus — APPROVED** [review]
Great updates, @johnbelamaric!
/lgtm
/approve
/hold cancel

**Comment by justaugustus** [comment]
Potential future update: It may be helpful if this guidance lived in a [more visible place](https://github.com/kubernetes/community/tree/master/contributors/devel) for contributors or [closer to the content](https://github.com/kubernetes/enhancements/tree/master/keps).

**Pull request merged**
